src/components/flat-table/sort/sort.component.js

```diff
@@ -33,7 +33,10 @@ const Sort = ({ children, onClick, sortType }) => {
       >
         {children}
         {sortType && (
-          <Icon type={sortType === "ascending" ? "sort_up" : "sort_down"} />
+          <Icon
+            type={sortType === "ascending" ? "sort_up" : "sort_down"}
+            color="--colorsUtilityMajor200"
```

Comment:
Given the value is always the same, I think this should be defined in `flat-table-row-header.style.js`. This would allow any icon to be used, not just the sort component.